web/src/components/documents/document-card-view-item/document-card-view-item.module.scss

```diff
@@ -2,7 +2,7 @@
     position: relative;
```

Comment:
Just noticed that the color of ticks was changed to grey (not sure that this PR affected this but maybe some previous)?
<img width="1088" alt="image" src="https://github.com/epam/badgerdoc/assets/13063442/d800bfb5-80f6-4ea2-a5f4-e3c57d7155e1">

On dev2 we have white ticks
<img width="1134" alt="image" src="https://github.com/epam/badgerdoc/assets/13063442/84cb8627-835b-4a9e-8752-c2d6e268d04f">

Reply:
fixed